### DISCUSSION TITLE
Unable to get Sepolia Eth for free! SEP-2024 everywhere eth on main is required

### DISCUSSION THREAD

**Discussion by Anon7777777** [Cyfrin/foundry-full-course-cu#2390] [Q&A] [answered]
Please I have tried every one of the Faucets except for PoW ones, every faucet requires a minimum of Real ETH on Mainnet !!!! please if someone could send enough SETH to go through the course I'd be very thankful

**Comment by Anon7777777** [comment]
couldn't try pow faucet because I couldn't login into gitcoin passport

**Comment by EngrPips** [comment]
Hey, the chainlink faucet does not require a minimum of Real ETH on Mainnet!!! https://faucets.chain.link/fuji

<img width="1440" alt="Screenshot 2024-09-04 at 21 41 49" src="https://github.com/user-attachments/assets/b0102832-d2e8-4236-b1fc-6993a1cc4d0b">

**Reply by Anon7777777** [reply]
![image](https://github.com/user-attachments/assets/d7f4e730-7a48-4f0c-a171-d2bd7473c61b)

 i have this problem on different browsers, not using a vpn

**Reply by EngrPips** [reply]
I don't know how to help with this. It worked fine for me without any issues. Maybe try to discover why your `request` doesn't look human.

**Reply by Anon7777777** [reply, marked as answer]
worked by switching to chrome based browser and using a wallet with some real money history

**Reply by EngrPips** [reply]
Good for you, congratulations.
